### PR TITLE
adding jupyter notebook usage code and IVIM analysis

### PR DESCRIPTION
**Pull request by merelvdthiel**
### Describe the changes you have made in this PR

_Adding of jupyter notebook and adaptations of readme to refer to the notebook_

### Link this PR to an issue [optional]

Fixes _#ISSUE-NUMBER_

### Checklist

- [ X ] Self-review of changed code
- [ ] Added automated tests where applicable
- [ ] Update Docs & Guides

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ If you would like to contribute with code, please follow the instructions below:
 *   [Guidelines for IVIM code contribution](doc/guidelines_for_contributions.md)
 *   [Guidelines to creating a test file](doc/creating_test.md)
```
Comment (oliverchampion) [inline README.md:19]: I think here we could start a header with 
## How to use
and give some explanation on how to look at the Git and how to use it. Also we could refer to the wrapper explanation that Ivan wrote too.

**Comment by oliverchampion** [comment]
Also, you have merged a .jpynb_checkpoints folder. Is this necessary or is this something that we can put on the ignore list?

**Comment by oliverchampion** [comment]
I see that the Notebook was run, with results in the commit. As these images will be stored in the Git history, and the history will grow every time someone changes the notebook, this can cause issues in the future. For code commits, we have told users not to run the jupyter notebook. One way of doing so can be seen here: https://gist.github.com/33eyes/431e3d432f73371509d176d0dfb95b6e. On the other hand, as this is an educational notebook it may be desirable that the output is visible. That said, I could only see the output after downloading and opening in my pycharm copy. This makes it hard for new users with little experience in Python. Possibly we could host the Jupyter notebook on GoogleColab instead, with all output visible and runnable in a browser. Let's discuss next session.

File: doc/Introduction_to_TF24_IVIM-MRI_CodeCollection_github_and_IVIM_Analysis_using_Python.ipynb [hunk 2]
```diff
@@ -0,0 +1,1107 @@
...
...tPf/qT5s6dq4EDB6qoqEhr1qxxLTZQWlrqtmjA8ePHNXv2bA0YMEDXXnutevbsqY0bN6pz587N/Sohr+4caMUL0uvNkQYAANAcvu7zAAAAWpNmzYnWmKqqKq1Zs0ZvvvmmVq9erczMTN1///2+eGufai1zZTS0iACLC4Sm3J9WVPX0N8uzl8jhNDRjVIIJlQEAarWWHqIl6PMAAEBr1tQeol1z3rSwsFDffPONfvGLX7j2Pf/885o3b56qqqo0btw4Pf3002rXrp2+/fbbllcPrxxOw2NQVrvtcPokG0WA2KwW5RQUS1KDoSgAAP5EnwcAANC4Zj3O+eCDD7qtgPnZZ5/p17/+tVJTUzVz5ky9/fbbys7Ols1mU7du3XxeLP5rRiN3mmWkxHPXUojJSImv9zgudxUCAAKJPg8AAKBxzboTraioSA899JBr++WXX1ZSUpKefvppSVJsbKzmzZun+fPn+7RIoC2oDcpyCoq1ZN0u1TicBGgAgIChzwMAAGhcs+5E+89//uOa8F+S3nvvPaWnp7u2L774Yu3bt8931QFtTEZKvGul1TCblQANABAw9HmAf+Q2svBXnr1EuT9N6QEACH7NCtGio6O1Z88eSVJNTY22bt2qSy65xHX8u+++02mnnebbCoE2JM9e4grQahxOVloFAAQMfR7gH7Vz357c19VO3WGzWkyqDADQXM16nHP06NGaOXOmHn30Ua1atUodOnTQiBEjXMc//fRT9e3b1+dFAm3ByXOg1W5L4o40AIDf0ecB/lF3yo7abea+BYDQ1Kw70R566CG1a9dOl19+uZ5++mk9/fTTCgsLcx1fvny5rrzySp8XCbR2nhopT4sNAADgL/7q85YuXaq4uDhFREQoKSlJmzdvbnT8woUL1a9fP7Vv316xsbGaMWOGjh8/3uzPBYJJ3b4uYdY7BGgAEKKadSda165dtWHDBlVUVKhjx46y2Wxux//+97+rY8eOPi0QaAscTsNjI1W77XAaZpQFAGhD/NHnvfLKK8rMzNSyZcuUlJSkhQsXKi0tTTt37lT37t3rjX/xxRc1c+ZMLV++XMOGDVNxcbFuueUWWSwW5eTknNL3A8yWkRLvWjyKuW8BIDRZDMNoU//feWVlpaKiolRRUaHIyEizywEAACGCHqL5kpKSdPHFF2vJkiWSJKfTqdjYWN19992aOXNmvf
```
Comment (etpeterson) [inline doc/Introduction_to_TF24_IVIM-MRI_CodeCollection_github_and_IVIM_Analysis_using_Python.ipynb:708]: I was able to run it in a colab with some changes up until this point. https://colab.research.google.com/drive/12FETqIZy2yOUceNwWDHca1b2OGZc1ATH?usp=sharing

**Comment by oliverchampion** [comment]
Hey! If you could add the notebook to .gitignore file such that it is not updated when people run it locally, than I can merge this request.

**Comment by merelvdthiel** [comment]
> Also, you have merged a .jpynb_checkpoints folder. Is this necessary or is this something that we can put on the ignore list?

We put it on the git ignore list now. 
The notebook itself we also put on the git ignore list

**Pull request merged**